core-blocks/shortcode/index.js

```diff
@@ -1,6 +1,7 @@
 /**
  * WordPress dependencies
  */
+import { removep, autop } from '@wordpress/autop';
```

Comment:
`core-blocks` does not have a dependency defined for `wp-autop`, so its existence is purely incidental and this could be prone to future breakage.